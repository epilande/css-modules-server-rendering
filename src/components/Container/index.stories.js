import React from 'react';
import { storiesOf } from '@kadira/storybook';
import Container from './';

storiesOf('Container', module)
  .add('default', () => (
    <Container>Hello World</Container>
  ))
  .add('centered with emoji', () => (
    <Container center>😀 😎 👍 💯</Container>
  ));
