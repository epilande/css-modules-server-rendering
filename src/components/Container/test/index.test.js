import React from 'react';
import { shallow } from 'enzyme';
import Container from '../index';

describe('<Container />', () => {
  let component;

  beforeEach(() => {
    component = shallow(
      <Container />,
    );
  });

  it('should render a Container', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render a div', () => {
    expect(component.find('div')).toHaveLength(1);
  });
});
