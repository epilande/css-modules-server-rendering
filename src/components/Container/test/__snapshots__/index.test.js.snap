exports[`<Container /> should render a Container 1`] = `
<div
  className="sc-bdVaJa kplsxE" />
`;
